local/network.go

```diff
@@ -965,6 +971,7 @@ func (ln *localNetwork) buildFlags(
 	// Flags for AvalancheGo
 	flags := []string{
 		fmt.Sprintf("--%s=%d", config.NetworkNameKey, ln.networkID),
+		fmt.Sprintf("--%s=%s", config.DataDirKey, dataDir),
```

Comment:
Bad formatting

Reply:
which one? don't understand

Reply:
nvm, this displayed incorrectly on my phone